tests/robot/DIRECTORY_TESTS/A.4_GET_DIRECTORY/A.4__d._EHR_does_not_exist.robot

```diff
@@ -33,12 +33,11 @@ Alternative flow 3: get directory on non-existent EHR
 
     create fake EHR
 
-    get DIRECTORY - fake ehr_id (JSON)
-
-    check response: is negative - EHR does not exist
+        Log  TO CLARIFY WITH @PPAZOS: use `GET /ehr/ehr_id/directory` OR `GET /ehr/ehr_id/directory/\${version_uid}` here?
```

Comment:
@ppazos

Reply:
@testautomation any will work, since it will fail because of the missing EHR